src/index.html

```diff
@@ -1,13 +1,49 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
   </head>
+
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+      src="./images/imac.jpeg"
+      alt="Product photo"
+      class="card__photo"
+      >
```

Comment:
remember about formatting, you can use prettier to do it 😉
```suggestion
      <img
        src="./images/imac.jpeg"
        alt="Product photo"
        class="card__photo"
      >
```